README.md

```diff
@@ -1,91 +1,91 @@
+### Installation
 
-* `jaqpot.login('username', 'password')`
+Install jaqpotpy using pip:
 
-Will login and set the api key that is needed.
-
-* `jaqpot.request_key('username', 'password')`
-
-Same as above you request the key and set it on jaqpot
-
-* `jaqpot.request_key_safe()`
-
-Will ask the user for the username and password by hidding the password if 
-jaqpot is used through a jupiter notebook etc
+```bash
+pip install jaqpotpy
+```
 
-### Set Key without login
+### Logging In
 
-Some users may have logged in through google or github. At the account page 
-a user can find an api key that can be used in order to have access on the services.
-These keys have short life and should be updated on each login.
+To use jaqpotpy, you need to log in to the Jaqpot platform. You can log in using your username and password or by setting an API key.
 
-* `jaqpot.set_api_key("api_key")`
+#### Login with Username and Password
 
-#### Get the key from user interface
+```python
+from jaqpotpy import Jaqpot
 
+jaqpot = Jaqpot()
+jaqpot.login('your_username', 'your_password') 
+```
 
-### Model training and deployment
+#### Request and Set API Key
 
+You can request an API key and set it:
+```python
+jaqpot.request_key('your_username', 'your_password')
+```
+or
+```python
+jaqpot.request_key_safe()  # Prompts for username and password securely
+```
+#### Set API Key Directly
 
-An example code that demonstrates a model deployemnt.
+If you already have an API key (you can retrieve one from https://app.jaqpot.org), you can set it directly:
 
+```python
+jaqpot.set_api_key("your_api_key")
+```
 
+### Model Training and Deployment
 
+Follow these steps to train and deploy your model on Jaqpot:
 
-*  Warning! One of the things that may differ from simpler training and validation routes is that you need to train your model with a pandas dataframe as input and not with Numpy arrays!
+	1. Train your model using pandas DataFrame as input.
+	2. Deploy the trained model using the deploy_on_jaqpot function.
 
+#### Example Code
+_Note: Ensure you use a pandas DataFrame for training your model._
 
 ```python
 from jaqpotpy import Jaqpot
 import pandas as pd
 from sklearn.linear_model import LinearRegression
 
+# Initialize Jaqpot
+jaqpot = Jaqpot()
 
+# Load your data
 df = pd.read_csv('/path/to/gdp.csv')
-lm = LinearRegression()
 
+# Train your model
+lm = LinearRegression()
 y = df['GDP']
 X = df[['LFG', 'EQP', 'NEQ', 'GAP']]
-
 model = lm.fit(X=X, y=y)
 
-jaqpot.deploy_sklearn(model, X, y, title="Title", description="Describe")
+# Deploy the model on Jaqpot
+jaqpot.deploy_sklearn(model, X, y, title="GDP Model", description="Predicting GDP based on various factors")
 ```
 
-The function will inform you about the model id that is created and is available through the user interface and the API.
-
-
-* Result
-
+The function will provide you with the model ID that you can use to manage your model through the user interface and API.
```

Comment:
@periklis91 shall we update this code example with a better example that uses `deploy_on_jaqpot` function?